custom_components/modbus_local_gateway/tcp_client.py

```
@@ -133,6 +133,7 @@ async def update_slave(
 
                     if (
                         modbus_response
+                        and isinstance(modbus_response, ReadRegistersResponseBase)
```

Ruff (0.8.2)

Undefined name `ReadRegistersResponseBase`

(F821)